### PR TITLE
feat(plugin): NoSentryConsole

### PR DESCRIPTION
**Pull request by WorriedArrow**
**NoSentryConsole**: Disables the sentry console log hook to return breadcrumbs to console. Very useful for development and reporting issues alike.

### The problem
Discord uses Sentry and it will install a hook into the console object.
Doesn't seem that bad, right?
Well, it makes the breadcrumbs/origin in the console from the file that installs that hook,
which ends up making it very hard to tell where non-error logs are coming from!

### The solution
So this plugin just deletes that hook.
The hook doesn't even do anything anyway, thanks to NoTrack.
This just removes that unnecessary code and allows for a much better experience when debugging,
developing plugins, or reporting issues!
What it does

### CODE REVIEW

File: src/plugins/noSentryConsole.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            // Disables Sentry console instrument and returns breadcrumbs to console
+            find: ".installedLogHooks)",
+            replacement: {
+                match: /if\(\w\.getDebugLogging\(\)&&!\w.installedLogHooks\)/,
+                replace: "if(false)"
+            }
+        },
+    ],
+
+    settingsAboutComponent() {
```
Comment (Vendicated) [inline src/plugins/noSentryConsole.tsx:39]: instead of using this, you should give your plugin a folder and add a README.md file

**Comment by Vendicated** [comment]
why not just add this to the NoTrack plugin instead?

File: src/plugins/noSentryConsole.tsx (hunk 1 above)
Comment (WorriedArrow) [reply src/plugins/noSentryConsole.tsx:39]: Will add to NoTrack.

File: src/plugins/_core/noTrack.ts [hunk 2]
```diff
@@ -51,6 +51,13 @@ export default definePlugin({
                     replace: "return;"
                 }
             ]
-        }
+        },
+        {
+            find: ".installedLogHooks)",
+            replacement: {
+                match: /if\(\w\.getDebugLogging\(\)&&!\w.installedLogHooks\)/,
```
Comment (Vendicated) [inline src/plugins/_core/noTrack.ts:58]:
```suggestion
                match: /if\(\i\.getDebugLogging\(\)&&!\i.installedLogHooks\)/,
```

File: src/plugins/_core/noTrack.ts [hunk 3]
```diff
@@ -51,6 +51,13 @@ export default definePlugin({
                     replace: "return;"
                 }
             ]
-        }
+        },
+        {
+            find: ".installedLogHooks)",
+            replacement: {
+                match: /if\(\i\.getDebugLogging\(\)&&!\i.installedLogHooks\)/,
```
Comment (Nuckyz) [inline src/plugins/_core/noTrack.ts:58]:
```suggestion
                match: /if\(\i\.getDebugLogging\(\)&&!\i\.installedLogHooks\)/,
```

**Review by Nuckyz — DISMISSED** [review]

**Comment by Vendicated** [comment]
looks good, thank you!

**Pull request merged**